bin/tictactoe

```diff
@@ -1,3 +1,42 @@
-#!/usr/bin/env ruby
```

Comment:
@Lani1321 @klovae  why are we removing this? this is to make the bin file know that you are using ruby code within the file.